tests/Integration/Behaviour/Features/Context/CartAwareTrait.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Tests\Integration\Behaviour\Features\Context;
+
+use Context;
+
+/**
+ * aims to provide access to cart through contexts
+ */
+trait CartAwareTrait
+{
+    public function getCurrentCart()
```

Comment:
overkill, more because `Context::getContext()->cart` can be called.

Reply:
this will be used when we will achieve to separate current cart and processing one